contrib/ansible/roles/kubernetes-addons/templates/kube-addons.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Kubernetes Addon Object Manager
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+
+[Service]
+ExecStart=/etc/kubernetes/kube-addons.sh
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
This service was brought over from the saltbase. They just changed how this happens based on #8107. I will look into what needs to change here.